### PR TITLE
JavaScript capstone project - Your API-based webapp

### PR DESCRIPTION
**Pull request by Nkaleth**
Objectives of this PR:

- Use JavaScript to make websites dynamic and build basic single page apps.
- Use ES6 syntax.
- Use ES6 modules.
- Use callbacks and promises.
- Use webpack.
- Apply JavaScript best practices and language style guides in code.
- Use AAA pattern for unit tests.
- Write units tests for a JavaScript app.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Use API documentation.
- Understand and use JSON.
- Make JavaScript code asynchronous.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
+      </div>
+    </header>
+    <main>
+      <section class="gallery"><!--gallery div --></section>
+      <div class="grid-container"></div>
+    </main>
+    <footer>
+      <p>Created by Microverse under CC license</p>
+    </footer>
+  </body>
```
Comment (Shedrack-Sunday) [inline dist/index.html:39]: - [ ] Good job making the footer with the right content. 👏 . However, you'll notice it's sectionalized with a line that I believe you missed. Kindly include it to give your web app a more appealing and clear-cut view from a user perspective 👍 
- [ ] Also, please adjust the font to  make it more visible 
# Your view:
image

# Wireframe:
image

File: dist/index.html [hunk 2]
```diff
@@ -1,34 +1,40 @@
...
+          src="https://raw.githubusercontent.com/fmanimashaun/artwork-listing-app/combined_soln/src/img/logo.png"
+          class="home-logo"
+          alt=""
+          sizes=""
+          srcset=""
+        />
+        <span class="fd-number">Gallery (<span class="fd-count">0</span>)</span>
+        <span class="fd-categories">Painting</span>
+        <span class="fd-categories">Shop</span>
+        <div class="artworks">
```
Comment (Shedrack-Sunday) [inline dist/index.html:27]: - [ ] You did great making the design, which I'll applaud you for, but I strongly suggest we take the view seriously from a user perspective:

- The text fonts look too small, and I think we can bold them a little for better user view 👍 
- From the wireframe, you can see how visible the popup is relative to the reflective background. I advise adjusting the saturation or making the popup background entirely white. again, this is for the best web accessibility 👍

## Your views:
image
 
 ## Wireframe with clarity:
 
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Nkaleth , @AyomipoMajek , and @francis3400   👋

CHANGES REQUIRED ♻️
Great work on the project 👏
You've done well implementing the requirements, However, there are a few you need to work on, to move to the next project. 💯


Highlights
✅All linters are checked 🔢 
✅ Descriptive commit messages 👍 

 

Required Changes ♻️

 - [ ] You did make effort in the Project and you really did a great job  👏 . However, there seem to be a few adjustments needed to be made, based on the design given, to make your whole code stand out and look good. 👍
 
 - [ ] The popups you implemented look good based on the functionality, and good job with that 👏 . However, the fonts for the text look very small. Also, the reflection from the homepage saturation does not sit well with your choice of `grey` and the color of the text. I've made a review on the reservation, but kindly make the same changes for comments. 👍 

-  Implementing web accessibility in website design and development benefits developers by increasing the usability and reach of the website, attracting a wider range of users and potential customers. It also demonstrates a commitment to inclusivity and responsible design practices, improving the developer's reputation and potentially reducing the risk of legal consequences. Furthermore, incorporating accessibility considerations can enhance the overall user experience and lead to more positive feedback and engagement.

 Kindly fix the margin on the right of the screen view. good job so far 👏

check the comment section for a detailed review

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: dist/index.html (hunk 1 above)
Comment (KDlamini) [reply dist/index.html:39]: Well resolved ✔️

File: dist/index.html (hunk 2 above)
Comment (KDlamini) [reply dist/index.html:27]: Well resolved ✔️

**Review by KDlamini — APPROVED** [review]
### Hi @Nkaleth , @AyomipoMajek , and @francis3400 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good API integration
✔️ ES6 and Webpack are used
✔️ Application works as expected
✔️ No linter errors

# Status: Approved ✔️ 

LeonardoInGatsby

## Optional Suggestions

- The user interface of the cards can be improved to enhance user experience(UX). The width of the cards is a little wide making the text and images look a bit stretched.

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**